### PR TITLE
fix(github-enterprise): configure gh enterprise full url via env var

### PR DESCRIPTION
**Pull request by aorinevo**

### CODE REVIEW

File: src/adapters/github.ts [hunk 1]
```diff
@@ -309,7 +309,10 @@ class GithubAdapter extends GitAdapter {
   }
 
   protected getRepositoryUrl(repo: IRepo): string {
-    return `git@${shepherdGitHubEnterpriseUrl}:${repo.owner}/${repo.name}.git`;
+    return (
+        `git@${shepherdGitHubEnterpriseUrl}:${repo.owner}/${repo.name}.git`
```
Comment (aorinevo) [inline src/adapters/github.ts:313]: Drop `.replace` call.

Comment (aorinevo) [reply src/adapters/github.ts:313]: Rename to gitHubEnterpriseUrl

**Comment by aorinevo** [comment]
> Do we need to add an example in readme as well for setting the url for github enterprise server , cloud?

Yes, I was actually thinking of placing that in the env var section but those changes are on another feature branch.

**Review by kavitha186 — APPROVED** [review]

File: src/adapters/github.ts (hunk 1 above)
Comment (aorinevo) [reply src/adapters/github.ts:313]: This needs to be base url, not full url (e.g., api.github.com vs. api.github.com/api/v3)

**Comment by aorinevo** [comment]
cc @taylorperkins

File: README.md [hunk 2]
```diff
@@ -23,10 +23,11 @@ Install the Shepherd CLI:
 npm install -g @nerdwallet/shepherd
 ```
 
-If using GitHub Enterprise, ensure the following environment variable is exported:
+If using GitHub Enterprise, ensure the following environment variable are exported:
 
 ```
-export SHEPHERD_GITHUB_ENTERPRISE_URL={company_github_enterprise_url} # e.g., github.test.com
+export SHEPHERD_GITHUB_ENTERPRISE_URL={company_github_enterprise_base_url} # e.g., api.github.com
```
Comment (aorinevo) [inline README.md:29]: This should be SHEPHERD_GITHUB_ENTERPRISE_BASE_URL.

**Pull request merged**